config/non-serializable/mailer.js

```diff
@@ -3,4 +3,5 @@ module.exports = {
     host: `${process.env.MAILER_HOST || 'localhost'}`,
     port: `${process.env.MAILER_PORT || 25}`,
   },
+  auth: `${process.env.MAILER_AUTH || '{}'}`,
```

Comment:
if ` is what I think [it is](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) it can't be used here because it always produces a string as output. The two possible cases [AFAIK](https://github.com/nodemailer/nodemailer/blob/33b62e2ea6bc9215c99a9bb4bfba94e2fb27ebd0/lib/smtp-transport/index.js#L88-L97) are:
- `{}` (empty dictionary or in any case a falsy value)
- `{ 'user': '...', 'pass': '...' }`